app/Http/Controllers/SampleController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Mail;
+use App\Mail\SampleNotification;
+
+class SampleController extends Controller
+{
+  public function SampleNotification()
```

Comment:
I think this file is not necessary because there is no line calling this function. I also tried commenting this out and the functionality still works. Please confirm.